### PR TITLE
Solvent column corrections

### PR DESCRIPTION
**Pull request by anujad95**
## Fixes
- Unit of convective heat flow corrected
- Convective and total inter phase heat transfer equations corrected

## Summary/Motivation:
Corrections have been made to the heat transfer equations in the solvent column model. This is expected to improve robustness.

## Changes proposed in this PR:
- Using average liquid and vapor phase temperatures in the convective heat transfer equation
- Using the appropriate sign conventions and phase properties for calculating the total rate of heat transfer

### Legal Acknowledgement

By contributing to this software project, I agree to the following terms and conditions for my contribution:

1. I agree my contributions are submitted under the license terms described in the LICENSE.txt file at the top level of this directory.
2. I represent I am authorized to make the contributions and grant the license. If my employer has rights to intellectual property that includes these contributions, I represent that I have received permission to make contributions and grant the required license on behalf of that employer.

### CODE REVIEW

File: idaes/generic_models/unit_models/column_models/solvent_column.py [hunk 1]
```diff
@@ -312,32 +312,32 @@ def _make_performance(self):
         self.diameter_column = Var(domain=Reals,
                                    initialize=0.1,
                                    units=pyunits.m,
-                                   doc='Column diameter')
+                                   doc='Column diameter (m)')
```
Comment (andrewlee94) [inline idaes/generic_models/unit_models/column_models/solvent_column.py:315]: I generally suggest not including the units in the doc string, as it presumes a specific set of units and effectively duplicates the information stored in units.  Thus, it becomes possible for the doc string to become inconsistent with the actual units if one s changed but not the other,

File: idaes/generic_models/unit_models/column_models/solvent_column.py [hunk 2]
```diff
@@ -622,38 +624,37 @@ def rule_heat_transfer_coeff_Ack(blk, t, x):
...
+        # Convective interphase heat transfer [J/s.m]
         @self.Constraint(self.flowsheet().time,
                          self.vapor_phase.length_domain,
-                         doc="heat transfer - vapor side ")
-        def vapor_phase_volumetric_heat_flux(blk, t, x):
+                         doc="Convective interphase heat transfer")
+        def vapor_phase_convective_heat_flow(blk, t, x):
             if x == self.vapor_phase.length_domain.first():
-                return blk.heat_flux_vap[t, x] == 0
+                return blk.heat_flow_vap[t, x] == 0
```
Comment (andrewlee94) [inline idaes/generic_models/unit_models/column_models/solvent_column.py:633]: This should be `Constraint.Skip`

File: idaes/generic_models/unit_models/column_models/solvent_column.py [hunk 3]
```diff
@@ -622,38 +624,37 @@ def rule_heat_transfer_coeff_Ack(blk, t, x):
...
             else:
                 zb = self.vapor_phase.length_domain.at(value(self.zi[x]) - 1)
-                return blk.heat_flux_vap[t, x] == blk.h_v_Ack[t, x] * \
-                    (blk.liquid_phase.properties[t, zb].temperature -
-                      blk.vapor_phase.properties[t, x].temperature)
+                return blk.heat_flow_vap[t, x] == blk.h_v_Ack[t, x] * \
```
Comment (andrewlee94) [inline idaes/generic_models/unit_models/column_models/solvent_column.py:636]: Why did you update this to use the average temperature between the two points? I believe this model is written explicitly using a finite element approach assuming that the state in each cell is equal to the value of the finite element. I.e. I believe the original form was correct.

**Review by andrewlee94 — CHANGES_REQUESTED** [review]
Mostly looks good, but I had a few comments. The most significant is asking why you changed the way the convective heat transfer is calculated.

File: idaes/generic_models/unit_models/column_models/solvent_column.py (hunk 1 above)
Comment (anujad95) [reply idaes/generic_models/unit_models/column_models/solvent_column.py]: Units have been removed from all doc strings, as suggested.

File: idaes/generic_models/unit_models/column_models/solvent_column.py (hunk 2 above)
Comment (anujad95) [reply idaes/generic_models/unit_models/column_models/solvent_column.py]: I agree, this change has been made.

File: idaes/generic_models/unit_models/column_models/solvent_column.py (hunk 3 above)
Comment (anujad95) [reply idaes/generic_models/unit_models/column_models/solvent_column.py:636]: I agree with this as well. I have changed it to the original equation.

**Review by andrewlee94 — APPROVED** [review]
Looks OK to me.

**Pull request merged**